docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -485,14 +498,48 @@
                 delay={{ boot_wait_time }}
                 timeout={{ wait_for_timeout }}
                 state=started
-            register: waiting_unlock
+            register: waiting_reboot
             retries: 40
             delay: 20
 
           # Waiting task after unlock to catch right status
           - wait_for:
               timeout: 450
               msg: Waiting after unlock due to apply DM config
+            register: waiting_after_reboot
+            failed_when: false
+
+          # Fix block: sometimes system reboot twice
```

Comment:
"Fix block" doesn't say anything. It seems to be related just for this commit.

nit: the system reboots

Reply:
Modified. Thanks!